.github/actions/test-pytorch-binary/action.yml

```diff
@@ -2,6 +2,12 @@ name: Test pytorch binary
 
 description: Pulls the docker image and tests the pytorch binary using it. All env variable referenced in the "Test PyTorch binary" step must be set in the GITHUB_ENV file
 
+inputs:
```

Comment:
I don't think you need this input because I can see `BUILD_ENVIRONMENT` env variable is there.  If you need to use it on the workflow, the syntax is `env.BUILD_ENVIRONMENT`, for example `if: always() && env.BUILD_ENVIRONMENT == 'linux-s390x-binary-manywheel'`